sql/encodings/utf16_bin.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encodings
+
+// Utf16_bin_RuneWeight returns the weight of a given rune based on its relational sort order from
+// the `utf16_bin` collation.
+func Utf16_bin_RuneWeight(r rune) int32 {
+	weight, ok := utf16_bin_Weights[r]
+	if ok {
+		return weight
+	} else if r >= 0 && r <= 55295 {
```

Comment:
What are all these magic numbers?

Reply:
Could use constants or comments

Reply:
For reference for anyone that checks out this PR later: these files are all generated from https://github.com/dolthub/collation-extractor. To understand them, it's easier to look at the related structures under the `rune_comparator.go` file than to look at a generated file.